CSS_VARIABLES.md

```diff
@@ -0,0 +1,295 @@
+  - <a href="#font-size">font-size</a>
+  - <a href="#line-height">line-height</a>
+  - <a href="#elevation">elevation</a>
+  - <a href="#spacing">spacing</a>
+  - <a href="#mini-unit">mini-unit</a>
+
+
+### Variables
+
+Use this to remove round corners or make them more round.
+
+`--round-corners: 3px;`
+
+### palette
+
+`--palette-divider: rgba(0, 0, 0, 0.12);`
+
+#### palette-primary
+
+Color palette that is used as the primary color.
+
+`--palette-primary-100: #EBF6FF;  /* Before 6.3.0: --palette-primary-lightest */`
+
+`--palette-primary-200: #B7DCFF;  /* Before 6.3.0: --palette-primary-lighter */`
+
+`--palette-primary-300: #61B3FF;  /* Before 6.3.0: --palette-primary-light */`
+
+`--palette-primary-400: #2897FF;  /* Before 6.3.0: --palette-primary-main */`
+
+`--palette-primary-500: #0070D9;  /* Before 6.3.0: --palette-primary-main */`
+
+`--palette-primary-600: #0062BE;  /* Before 6.3.0: --palette-primary-main */`
+
+`--palette-primary-700: #005AAE;  /* Before 6.3.0: --palette-primary-main */`
+
+`--palette-primary-800: #004688;  /* Before 6.3.0: --palette-primary-dark */`
+
+`--palette-primary-900: #00386D;  /* Before 6.3.0: --palette-primary-darkest */`
+
+#### palette-background
+
+Color palette that is used for background colors.
+
+`--palette-background-body: #FFFFFF;`
+
+`--palette-background-popover: #FFFFFF;`
+
+`--palette-background-tooltip: #65696B;`
+
+`--palette-background-input: #FFFFFF;`
+
+`--palette-background-input-disabled: #EFEFEF;`
+
+#### palette-text
+
+Color palette that is used for text.
+
+`--palette-text-000: #FFFFFF;  /* Before 6.3.0: --palette-text-light */`
+
+`--palette-text-100: #65696B;  /* Before 6.3.0: --palette-text-secondary */`
+
+`--palette-text-500: #353F44;  /* Before 6.3.0: --palette-text-primary */`
+
+`--palette-text-900: #14171A;  /* Before 6.3.0: --palette-text-dark */`
+
+`--palette-text-placeholder: #9FA4A6;  /* Before 6.3.0: --palette-grey-lighter */`
+
+`--palette-text-input-disabled: #9FA4A6;  /* Before 6.3.0: --palette-grey-lighter */`
+
+#### palette-grey
+
+Color palette that is used for grey shades.
+
+`--palette-grey-100: #F4F7F7;  /* Before 6.3.0: --palette-grey-lightest */`
+
+`--palette-grey-200: #EAEFF0;  /* Before 6.3.0: --palette-grey-lightest */`
+
+`--palette-grey-300: #CBD1D2;  /* Before 6.3.0: --palette-grey-lighter */`
+
+`--palette-grey-400: #9FA4A6;  /* Before 6.3.0: --palette-grey-lighter */`
+
+`--palette-grey-500: #65696B;  /* Before 6.3.0: --palette-grey-main */`
+
+`--palette-grey-600: #49545C;  /* Before 6.3.0: --palette-grey-dark */`
+
+`--palette-grey-700: #32404D;  /* Before 6.3.0: --palette-grey-darkest */`
+
+`--palette-grey-800: #202E3E;  /* Before 6.3.0: --palette-grey-darkest */`
+
+`--palette-grey-900: #132033;  /* Before 6.3.0: --palette-grey-darkest */`
+
+#### palette-negative
+
+Color palette that is used for indicating something negative and is usually red.
+
+`--palette-negative-100: #FCE5D9;  /* Before 6.3.0: --palette-error-lightest */`
```

Comment:
The use of `negative` and `positive` is inconsistent with existing verbiage around colours. The proximity to numbers is especially confusing (i.e.`negative 100`).

Please revert to `error`/`success` naming.